### ISSUE TITLE
How to report script error :) 

### ISSUE THREAD

**Issue by Meme01** [Secuter/SecuterYGOCustomCards#1]
I found a script error in card tarakasura 12340911 and retrofiting darkness 12341519. I hope you can fix it in a future update. Btw i love you custom card, Thanks :)

**Comment by Secuter** [comment]
Hi,
I fixed the Quick effect that didn't work on Tarakasura and the script error that was appearing in some Armor cards (including retrofitting darkness).
Let me know if these were the errors you meant and if it works now!
Ps: For Tarakasura did you have an error in the log or was it just the effect not working? Maybe if next time you can be more specific it can help me understand if I have identified the right error.
Thank you for reporting them. :D

**Comment by Meme01** [comment]
Hi, thanks for fixing the bug on the card. The 2 cards have worked fine now :) . I still found some bugs on other cards. Maybe later I will report here after I list the cards that have bugs. Thanks Secuter :D

**Comment by Secuter** [comment]
Hello, they should be fixed now.
- Double weapon had the wrong labels when choosing the effect, the effect itself already worked for me.
- Cracking codetalker, fixed now, there was a problem with the UpdateAttack function with other Cracking cards as well.
- Cracking cannonade, this should already work, to activate it requires that you control a Cracking monster with at least 500 atk and that the opponent has a Spell/Trap

**Comment by Meme01** [comment]
Thanks you very much bro :)
I found bug 12341503 - granite Armor golem. Bug on effect cant special summon attached monster in this card. "parameter 6 should be int"

By the way if you are also busy with your life or work it's fine if you can't fix the bug that I reported right away. I'm afraid my report will burden you. I hope you are always healthy there :)

**Comment by Secuter** [comment]
Hi, don't worry too much, I already look at this when I have time or when I'm creating/testing new cards.
So far I have solved these bugs quickly because they were mostly oversight that are fast to solve once I know what card to check :)

**Comment by Meme01** [comment]
Hehe Thanks Secuter :D

**Comment by Meme01** [comment]
Hai
I found bug

12345020 cracking chain - The effect to reduce the monster's atk doesn't work

**Comment by Secuter** [comment]
> Hai I found bug
> 
> 12345020 cracking chain - The effect to reduce the monster's atk doesn't work

Hi, it had the location for the effect LOCATION_FZONE (the one for the Field zone) instead of LOCATION_SZONE, now I fixed it.
I also checked the other Fields/Continuous and fixed the ones that had the wrong location. :D

**Comment by Meme01** [comment]
All lvl 12 divine beast cards can still be targeted by enemy card effects. I don't know if this is a bug or just a typo in the text card effect

**Comment by CutBoard777** [comment]
Hi, it seems that Wing Cracking Dragon's negate does not work at all. Also, the restriction of Cracking Gunner to summon him only once per turn seems to not work (Cracking Agent works fine though).

**Comment by Secuter** [comment]
> All lvl 12 divine beast cards can still be targeted by enemy card effects. I don't know if this is a bug or just a typo in the text card effect
I did some tests but they seem to be correctly not targettable.
In what situation were they targettable? Maybe it's the interaction with another card that has a bug.

**Comment by Secuter** [comment]
> Hi, it seems that Wing Cracking Dragon's negate does not work at all. Also, the restriction of Cracking Gunner to summon him only once per turn seems to not work (Cracking Agent works fine though).

Fixed now 😃
